scripts/util/metadata/batching_sink.rb

```diff
@@ -124,7 +124,7 @@ def initialize(hash)
       "retry_backoff_secs" =>
```

Comment:
@binarylogic this needs to be renamed to `retry_initial_backoff_secs`

Reply:
Done